recommendation-training/train-workflow.py

```diff
@@ -244,6 +277,7 @@ def _sanitize_categorical(x, num_embeddings: int, name: str):
             "user_items",
             "item_features",
             "item_textual_features_embed",
+            "item_name_features_embed",
```

Comment:
The `item_name_features_embed` should not be in this list of feature_views. It does not need to be materialized  since item_name_features_embed  is fed via PushSource (since you push embeddings directly to the ONLINE store using store.push). In fact, `item_embedding, user_items, item_textual_features_embed `, they’re push-only; we push them online directly and hence can be removed from the `feature_views`